docs/commands/ft.create.md

```diff
@@ -97,23 +92,29 @@ is document attribute that you use as a binary safe payload string to the docume
 <details open>
 <summary><code>MAXTEXTFIELDS</code></summary> 
 
-forces RediSearch to encode indexes as if there were more than 32 text attributes, which allows you to add additional attributes (beyond 32) using `FT.ALTER`. For efficiency, RediSearch encodes indexes differently if they are
-  created with less than 32 text attributes.
+forces RediSearch to encode indexes as if there were more than 32 text attributes, which allows you to add additional attributes (beyond 32) using `FT.ALTER`. For efficiency, RediSearch encodes indexes differently if they are created with less than 32 text attributes.
 </details>
 
 <details open>
 <summary><code>NOOFFSETS</code></summary> 
 
-does not store term offsets for documents. It saves memory, but does not
-  allow exact searches or highlighting. It implies `NOHL`.
+does not store term offsets for documents. It saves memory, but does not allow exact searches or highlighting. It implies `NOHL`.
 </details>
 
 <details open>
 <summary><code>TEMPORARY</code></summary> 
 
 creates a lightweight temporary index that expires after a specified period of inactivity. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore, you should consider using `SKIPINITIALSCAN` to avoid costly scanning.
```

Comment:
I suggest to also mention here the period is in seconds, although it is specified in the above syntax `[TEMPORARY seconds]`

Reply:
@oshadmi added seconds